Strapi API.postman_collection.json

```diff
@@ -0,0 +1,192 @@
+{
```

Comment:
It will be nice to explain how to add new tests and modify existing ones.

Reply:
yeah, I'll do that. Also, one thing I was wondering was if all tests would be run every pr. For example, if I create a test for each pr then all the tests from previous prs will also be run unless we delete the tests.

Reply:
I think we should run all tests to make sure there is no regression.